google/cloud/internal/oauth2_authorized_user_credentials.h

```diff
@@ -48,9 +48,9 @@ StatusOr<AuthorizedUserCredentialsInfo> ParseAuthorizedUserCredentials(
 
 /// Parses a refresh response JSON string into an authorization header. The
 /// header and the current time (for the expiration) form a TemporaryToken.
```

Comment:
"TemporaryToken"?

Reply:
Fixed here and anywhere else I found in this directory.